src/main/java/laddergame/view/LadderForm.java

```diff
@@ -0,0 +1,58 @@
+package laddergame.view;
+
+import laddergame.domain.Line;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public enum LadderForm {
+    LADDER_ROW_FILLED("-----"),
+    LADDER_ROW_EMPTY("     "),
+    LADDER_COL("|");
+
+    private final String unit;
+
+    LadderForm(String unit) {
+        this.unit = unit;
+    }
+
+    public static String joinUnitsFrom(final List<String> names, final List<Line> lines) {
+        return joinNames(names) + joinRows(lines);
+    }
+
+    private static String joinNames(final List<String> names) {
+        String joined = names.stream().map(name -> String.format("%6s", name)).collect(Collectors.joining());
+        return joined + System.lineSeparator();
+    }
+
+    private static String joinRows(final List<Line> lines) {
+        final StringBuilder ladderBuilder = new StringBuilder();
+        lines.forEach(line -> {
+            ladderBuilder.append(LADDER_ROW_EMPTY.unit);
+            ladderBuilder.append(LADDER_COL.unit);
+            joinRowUnits(ladderBuilder, line);
+            ladderBuilder.append(System.lineSeparator());
+        });
+        return ladderBuilder.toString();
+    }
+
+    private static void joinRowUnits(final StringBuilder ladderBuilder, final Line line) {
+        line.getPoints().forEach(point -> {
+            appendRowFilled(ladderBuilder, point);
+            appendRowEmpty(ladderBuilder, point);
+            ladderBuilder.append(LADDER_COL.unit);
+        });
+    }
+
+    private static void appendRowEmpty(final StringBuilder ladderBuilder, final Boolean point) {
```

Comment:
1. point라는 단어는 점수나 쇼핑몰에서의 포인트같은 정수 개념으로 이해되는데, 다른 변수명을 고려해보면 어떨까요? (생각해보니 '점'으로 볼 수도 있으나, Boolean 타입은 true/false를 나타내므로 주로 isXXX 네이밍을 많이 씁니다)
2. Boolean과 boolean의 차이가 무엇일까요? 그리고 boolean이 아닌 Boolean을 사용하신 이유를 코멘트로 남겨주세요!

Reply:
1. `point`라는 변수명을  `isRowFilled` 로 변경했습니다. 😄
제이온이 말씀해주신 것처럼 현재 상황에서 point가 어떤 의미를 가지는지 모르겠습니다.
(Line 클래스 필드에 List<Boolean> points 라고 선언했어서 point라는 네이밍을 했습니다.)
isRowFilled 와 같은 변수명을 이용해서 더 명확하게 표현하겠습니다.
```suggestion
    private static void appendRowEmpty(final StringBuilder ladderBuilder, final Boolean isRowFilled) {
```

Line 클래스에 있는 `List<Boolean>` 필드의 경우에는 `List`이므로 `isFilledPoints` 라는 네이밍이 아닌 `points`가 맞다고 생각이 들어 수정하지 않았습니다. 
```java
public class Line {
    private final List<Boolean> points;
    // private final List<Boolean> isFilledPoints;
```

2. Boolean과 boolean

`boolean`을 감싸준 Wrapper 클래스 `Boolean` 입니다.
Boolean을 이용하면 null 체크를 해줄 수 있다는 장점이 있지만 사용할 때 Wrapper를 벗겨주는 비용이 발생합니다.
현재 상황에서 null이 들어올 수 있는 상황은 아니라고 판단해서 boolean으로 수정하겠습니다. 🙂
Boolean을 사용한 이유는 조급한 마음에 생각하지 못한거 같습니다. 
보다 의미 있는 코드를 작성하도록 꼼꼼하게 보겠습니다! 😅

Reply:
1. 변수명이 더욱 직관적이게 되었네요~ 저도 `List<Boolean>`은 points로 두어도 될거같다는 생각이 드네요. Line 안에 point(점)이 있는건 자연스러우니까요!
2. Boolean을 이용하면 null 체크가 가능하다고 하셨는데, primitive 타입은 null 자체가 들어오는 것이 불가능합니다. 그리고 시간이 되신다면 박싱과 언박싱, 그리고 Wrapper 클래스에 관해 학습해 보시면 도움이 될 것 같습니다 :)